### PR TITLE
add configuration for S3 deployment 

### PR DESCRIPTION
**Pull request by petarblazevski**

### CODE REVIEW

**Comment by petarblazevski** [comment]
Ignore the merge branch. Problems with the rebase

File: README.md [hunk 1]
```diff
@@ -23,3 +23,36 @@ or
...
+
+This command will build the application in production mode.
+This means it will read the `.env.production` as primary env file, and fallback to `.env`
+
+After the build is complete, run the following command
+
+`yarn deploy`
+
+This will deploy the application to S3 and invalidate the CloudFront cache, but only
+if this is enabled
```
Comment (IAfanasovMob) [inline README.md:42]: `this` - cache?

File: package.json [hunk 2]
```diff
@@ -28,6 +26,7 @@
     "google-maps-api-loader": "^1.1.1",
     "iban": "0.0.14",
     "vue": "^2.6.11",
+    "vue-cli-plugin-s3-deploy": "^4.0.0-rc3",
```
Comment (IAfanasovMob) [inline package.json:29]: `rc` looks a bit scary 🙃 I don't mind to use it 🙃

File: vue.config.js [hunk 3]
```diff
@@ -22,8 +23,31 @@ module.exports = {
...
+    s3Deploy: {
+      registry: undefined,
+      awsProfile: 'default',
+      overrideEndpoint: false,
+      region: 'eu-west-1',
+      bucket: 'tegoedje-dev',
+      createBucket: true,
+      staticHosting: true,
+      staticIndexPage: 'index.html',
+      staticErrorPage: 'index.html',
```
Comment (IAfanasovMob) [inline vue.config.js:42]: do we need a task for proper `staticErrorPage`?

Comment (petarblazevski) [reply vue.config.js:42]: It would be good. These options are telling S3 to redirect the errors to Vue. But we will need a page for 404 for example

Comment (IAfanasovMob) [reply vue.config.js:42]: https://github.com/I-OWE-YOU/web-frontend/issues/122

**Pull request merged**